model/trace/rpc.yaml

```diff
@@ -5,50 +5,10 @@ groups:
     brief: 'This document defines semantic conventions for remote procedure calls.'
     events: [rpc.message]
     attributes:
-      - id: system
+      - ref: rpc.system
         requirement_level: required
-        brief: 'A string identifying the remoting system. See below for a list of well-known identifiers.'
-        type:
-          allow_custom_values: true
-          members:
-            - id: grpc
-              value: 'grpc'
-              brief: 'gRPC'
-            - id: java_rmi
-              value: 'java_rmi'
-              brief: 'Java RMI'
-            - id: dotnet_wcf
-              value: 'dotnet_wcf'
-              brief: '.NET WCF'
-            - id: apache_dubbo
-              value: 'apache_dubbo'
-              brief: 'Apache Dubbo'
-            - id: connect_rpc
-              value: 'connect_rpc'
-              brief: 'Connect RPC'
-      - id: service
-        type: string
-        requirement_level: recommended
```

Comment:
`requirement_level` is missing here and in `method`?

Reply:
Recommended is the default already, so it's fine to omit.